index.ipynb

```diff
@@ -82,7 +82,21 @@
     "\n",
     "Alternatively, you can also download individual notebook files (with the extension `.ipynb`) and open them in IPython.\n",
     "Note that some exercises make use of additional files (audio files etc.) which you'll then also have to download manually.\n",
-    "\n"
+    "\n",
+    "### Anconda Usage\n",
+    "\n",
+    "If we want to utilize the [Anaconda](https://www.anaconda.com/products/distribution) distribution platform for Python, we can create a dedicated environment `mycomac` (we can also use a more creative name) for this course.\n",
+    "\n",
+    "For summer term 2022, we use\n",
+    "\n",
+    "- `conda --version`: 4.12.0\n",
+    "- `conda-build --version`: 3.21.8\n",
+    "- `conda create -n mycomac python=3.9 pip numpy sympy scipy matplotlib notebook ipykernel jupyterlab ipympl pydocstyle pycodestyle autopep8 flake8`\n",
+    "- or enforcing exact version numbers as `conda create -n mycomac python=3.9.12 pip=22.0.4 numpy=1.22.3 sympy=1.10.1 scipy=1.8.0 matplotlib=3.5.1 notebook=6.4.10 ipykernel=6.13.0 jupyterlab=3.3.4 ipympl=0.9.0 pydocstyle=6.1.1 pycodestyle=2.8.0 autopep8=1.6.0 flake8=4.0.1`\n",
```

Comment:
I you want, you can use those versions in `environment.yml`, no need to confuse the students with that.

Reply:
Yes, in the `environment.yml` file I decided to use the version numbers from the environment I currently use to maintain the notebooks in order to have the version numbers somehow tracked.
Though, the `.binder` folder configuration is without explicit version requirement (except sfs), to have it more generic.
Does that make sense?